apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -55,19 +68,31 @@ const styles = {
   },
   body: {
     backgroundColor: 'white',
-    overflowY: 'scroll',
     paddingLeft: 10,
     paddingRight: 10,
     position: 'absolute',
     top: HEADER_HEIGHT,
     bottom: 0,
     left: 0,
-    right: 0
+    right: 0,
+    overflow: 'scroll'
```

Comment:
i'm assuming this is intentional, but want to double-check -- previously, we only had `overflowY` scrolling, but do we now want both X and Y to be scrollable? i would think we want the content to always fit horizontally instead of having a horizontal scrollbar

Reply:
Makes sense - I'll try putting it back